src/modules/rating/controllers/dtos/delete-rating.dtos.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger'
+
+export class DeleteRatingResponseDTO {
+	@ApiProperty({
+		description: 'ID of the deleted rating',
+	})
+	deletedRatingId: string
```

Comment:
Why do we need to return **deleteRatingId**?

Reply:
### to show the Message during success delete in return new DeleteRatingResponseDTO ![Screenshot 2024-03-20 103712](https://github.com/patecan-io-software/trn-ecom-phucuong-inventory/assets/83982760/343e63e1-9d17-4028-b544-993c3bb97c1f) like this 
![image](https://github.com/patecan-io-software/trn-ecom-phucuong-inventory/assets/83982760/1681395c-fd90-4c36-a6e2-937f3d4b61e0)